Keyboard/DefaultKeyboard.swift

```diff
@@ -39,15 +39,7 @@ func defaultKeyboard() -> Keyboard {
     keyModeChangeNumbers.toMode = 1
     defaultKeyboard.add(key: keyModeChangeNumbers, row: 3, page: 0)
     
-    if #available(iOSApplicationExtension 11.0, *) {
-        if (KeyboardViewController.shared?.needsInputModeSwitchKey == true) {
-            let keyboardChange = Key(.keyboardChange)
-            defaultKeyboard.add(key: Key(keyboardChange), row: 3, page: 0)
-            defaultKeyboard.add(key: Key(keyboardChange), row: 3, page: 1)
-            defaultKeyboard.add(key: Key(keyboardChange), row: 3, page: 2)
-        } else if (KeyboardViewController.shared?.needsInputModeSwitchKey == false) {
-        }
-    } else {
```

Comment:
This else was referring to #available(iOSApplicationExtension 11.0, *)
so in case that was false the key was added anyways.

Reply:
There us no fallback specified for older versions than iOS 11 to provide the key if we remove this...